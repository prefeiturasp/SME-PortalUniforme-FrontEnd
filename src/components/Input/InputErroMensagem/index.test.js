import { mount } from "enzyme";
import React from "react";
import { InputErroMensagem } from "./index";

describe("test <InputErroMensagem>", () => {
  let wrapper;
  beforeAll(() => {
    wrapper = mount(<InputErroMensagem />);
  });

  it("renders component", () => {
    expect(wrapper).toMatchSnapshot();
  });
});
