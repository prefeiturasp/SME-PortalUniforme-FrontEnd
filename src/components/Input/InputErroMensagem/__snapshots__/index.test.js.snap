// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test <InputErroMensagem> renders component 1`] = `
<InputErroMensagem>
  <div
    className="error-or-warning-message"
  />
</InputErroMensagem>
`;
